nixos/modules/services/misc/mautrix-whatsapp.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, config, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.mautrix-whatsapp;
+in
+{
+  options.services.mautrix-whatsapp = {
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Whether to enable mautrix-whatsapp, a puppeting bridge between Matrix and WhatsApp.
+
+        See <link xlink:href="https://github.com/tulir/mautrix-whatsapp/wiki/Bridge-setup">here</link> for documentation.
+     '';
+    };
+
+    config = mkOption {
+      type = types.lines;
```

Comment:
expect an attrset and convert to json, cf. https://github.com/NixOS/rfcs/pull/42

Reply:
@Ma27 have you an idea to integrate generated number (as_token and hs_token) to this type of configuration ?

Reply:
I have to admit that until now I've simply included some config files in a `mautrix-whatsapp.service`. I guess that there's no standardized approach to include/merge multiple yaml files.

Unfortunately I can't think of a better solution right now, so I guess that the best approach is to request such a feature at upstream (and probably create a patch for this).

Reply:
And what if I use `runCommand` to generate it on build and store config and registration file in the nix store ? Is this a good idea ? There is some guidelines on this tricks ?